src/hotspot/cpu/x86/x86.ad

```diff
@@ -5775,6 +5785,32 @@ instruct evminmaxFP_reg_eavx(vec dst, vec a, vec b, vec atmp, vec btmp, kReg ktm
   ins_pipe( pipe_slow );
 %}
 
+// --------------------------------- Signum ---------------------------
+
+instruct signumF_reg(regF dst, regF zero, regF one, rRegP scratch, rFlagsReg cr) %{
+  predicate(UseSSE>=1);
```

Comment:
Predicate looks redundant since you already added a check in match_rule_supported